### PR TITLE
adds int16 array support

### PR DESCRIPTION
**Pull request by JoanVicens**
Adds support int16 arrays.

### CODE REVIEW

File: src/Npgsql.FSharp.fs [hunk 1]
```diff
@@ -227,6 +227,7 @@ module Sql =
             | SqlValue.Bool boolean -> add boolean NpgsqlDbType.Boolean
             | SqlValue.Decimal number -> add number NpgsqlDbType.Numeric
             | SqlValue.Money number -> add number NpgsqlDbType.Money
+            | SqlValue.Long number -> add number NpgsqlDbType.Smallint
```
Comment (Zaid-Ajaj) [inline src/Npgsql.FSharp.fs:230]: This should be `SqlValue.Short` instead of `SqlValue.Long`, right?

**Review by Zaid-Ajaj — CHANGES_REQUESTED** [review]
The PR looks good! Just a minor remark here 😄

File: src/Npgsql.FSharp.fs (hunk 1 above)
Comment (JoanVicens) [reply src/Npgsql.FSharp.fs:230]: Yes, the copy-paste has betrayed me 😿

Comment (JoanVicens) [reply src/Npgsql.FSharp.fs:230]: ``SqlValue.Short``  was already in the match expression in line 222, so I will remove the line. Sounds good?

Comment (Zaid-Ajaj) [reply src/Npgsql.FSharp.fs:230]: That is even better, thanks!

**Review by JoanVicens — COMMENTED** [review]
changes where made

**Comment by Zaid-Ajaj** [comment]
@JoanVicens Just published the new version of Npgsql.FSharp and it includes your additions 🚀 😄 thanks a lot for the contribution 🙏

**Pull request merged**